packages/data-bite/src/components/EditorPanel.js

```diff
@@ -346,7 +348,7 @@ const EditorPanel = memo(() => {
                     <li className="three-col">
                       <TextField value={config.dataFormat.prefix} section="dataFormat" fieldName="prefix" label="Prefix" updateField={updateField} />
                       <TextField value={config.dataFormat.suffix} section="dataFormat" fieldName="suffix" label="Suffix" updateField={updateField} />
-                      <TextField type="number" value={config.dataFormat.roundToPlace} section="dataFormat" fieldName="roundToPlace" label="Round" updateField={updateField} />
+                      <TextField  type="number" value={config.dataFormat.roundToPlace} section="dataFormat" fieldName="roundToPlace" label="Round" updateField={updateField} min='0' max='99'  />
```

Comment:
@Atash3000 - nice addition. We probably don't even need it up to 99 but we'll leave it as is for now until somebody tells us to limit decimals. Good work!